src/blocks/CaseStudyParallax/index.tsx

```diff
@@ -0,0 +1,307 @@
+'use client'
+import * as React from 'react'
+
+import { BackgroundGrid } from '@components/BackgroundGrid'
+import { BackgroundScanline } from '@components/BackgroundScanline'
+import { BlockSpacing } from '@components/BlockSpacing'
+import { Button } from '@components/Button'
+import { Gutter } from '@components/Gutter'
+import { Media } from '@components/Media'
+import { QuoteIconAlt } from '@root/icons/QuoteIconAlt'
+import { Page } from '@root/payload-types'
+import { useResize } from '@root/utilities/use-resize'
+
+import classes from './index.module.scss'
+
+type Props = Extract<Page['layout'][0], { blockType: 'caseStudyParallax' }>
+
+type StickyBlockProps = Props & {
+  currentIndex: number
+}
+
+type QuoteProps = {
+  card: any
+  isVisible?: boolean
+  className?: string
+}
+
+export const QuoteBlock: React.FC<QuoteProps> = props => {
+  const { isVisible, card, className } = props
+  return (
+    <div
+      className={[isVisible && classes.isVisible, 'cols-8 grid', className]
+        .filter(Boolean)
+        .join(' ')}
+      aria-hidden={!isVisible}
+    >
+      <QuoteIconAlt className={classes.quoteIcon} />
+      <div
+        aria-hidden={!isVisible}
+        className={[classes.quote, 'cols-16'].filter(Boolean).join(' ')}
+      >
+        {card.quote}”
+      </div>
+
+      <div
+        aria-hidden={!isVisible}
+        className={[classes.authorWrapper, 'cols-16'].filter(Boolean).join(' ')}
+      >
+        <div className={classes.media}>
+          {typeof card.logo !== 'string' && <Media resource={card.logo} />}
+        </div>
+        <div className={classes.author}>{card.author}</div>
+      </div>
+
+      {typeof card.caseStudy !== 'string' && (
+        <div className={['cols-8 cols-m-4 cols-s-8'].filter(Boolean).join(' ')}>
```

Comment:
@paulpopus Do you prefer the button only spanning `4` columns in mid break or should it be spanning the full `8` columns?

Open to your thoughts here.

Here is what both look like:
![Screen Shot 2024-01-25 at 12 19 13 PM](https://github.com/payloadcms/website/assets/35232443/a840bf26-d5b8-4182-9d44-2fa13f95a30c)
![Screen Shot 2024-01-25 at 12 19 24 PM](https://github.com/payloadcms/website/assets/35232443/d2337a44-c8da-4a90-957b-81697f2d527d)